server/controllers/premisesController.test.ts

```diff
@@ -36,18 +36,21 @@ describe('PremisesController', () => {
     it('should return the premises detail to the template', async () => {
       const premises = { name: 'Some premises', summaryList: { rows: [] as Array<SummaryListItem> } }
       const bookings = createMock<GroupedListofBookings>()
+      const currentResidents = createMock<Array<{ text: string; html: string }>>()
```

Comment:
Could we do `createMock<Array<TableRow>>` here?

Reply:
Yes